### DISCUSSION TITLE
Party size limit?

### DISCUSSION THREAD

**Discussion by Snjbrc** [diasurgical/DevilutionX#4512] [Q&A] [answered]
I just want to ask what the party size limit is because I can't find the answer by googling it. Is it limited to 4 players or more? Can it be increased? How?

**Comment by qndel** [comment, marked as answer]
4

**Reply by Snjbrc** [reply]
Can it be increased in any way?

**Reply by AJenbo** [reply]
You can do so by editing the code and compiling a build for that. Some glitches will happen, and it can lead to crashes (thought it was somewhat stable when I tested), since spawn locations, chat and town portals are only setup to handle 4 players.

**Reply by Tormentor667** [reply]
But can't this be changed somehow engine-side to make it handle 8 or more players?

**Reply by DakkJaniels** [reply]
> But can't this be changed somehow engine-side to make it handle 8 or more players?

Sure, anything is possible

**Reply by Tormentor667** [reply]
So are you going to implement it? :)

**Reply by StephenCWills** [reply]
Nobody is working on it right now, that we know of.

**Reply by DakkJaniels** [reply]
> So are you going to implement it? :)

No, are you?

**Reply by Tormentor667** [reply]
Excuses, I thought you are a developer :)
